src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -110,6 +110,12 @@ public NodeSettings()
         /// <summary>Whether use of checkpoints is enabled or not.</summary>
         public bool UseCheckpoints { get; set; }
 
+        /// <summary>
+        /// If this block is in the chain assume that it and its ancestors are valid and skip their script verification. 
+        /// Null to not assume valid blocks and therefore validate all blocks.
+        /// </summary>
+        public uint256 BlockAssumedValid { get; set; }
```

Comment:
Is there not a ConsensusSettings?
I would expect `BlockAssumedValid` to go there.

Reply:
No ConsensusSettings but I could create one. What else would be in it? There is a ConsensusOptions class but it doesn't seem to be initialized by config.

Reply:
yeah create one, basically anything that is runtime configurable should go in settings, consensus options is part of the consensus rules so its a different thing altogether.

Reply:
Sounds good. Working on it...